Mandir creation fails

```
DESTDIR=/tmp/tinyssh-release make -C /tmp/tinyssh-build install
make[1]: Entering directory '/tmp/tinyssh-build'
sh -e make-install.sh /tmp/tinyssh-release
=== Wed Jul 27 14:44:12 UTC 2016 === installing bin directory /tmp/tinyssh-release/usr/bin
=== Wed Jul 27 14:44:13 UTC 2016 ===   installing /tmp/tinyssh-build/build/bin/tinysshd-makekey -> /tmp/tinyssh-release/usr/bin/tinysshd-makekey
=== Wed Jul 27 14:44:13 UTC 2016 ===   installing /tmp/tinyssh-build/build/bin/tinysshd-printkey -> /tmp/tinyssh-release/usr/bin/tinysshd-printkey
=== Wed Jul 27 14:44:13 UTC 2016 ===   installing /tmp/tinyssh-build/build/bin/tinysshd -> /tmp/tinyssh-release/usr/bin/tinysshd
=== Wed Jul 27 14:44:13 UTC 2016 === finishing
=== Wed Jul 27 14:44:13 UTC 2016 === installing man directory /tmp/tinyssh-release/usr/share/man
_tinysshd-install: fatal: unable to stat directory /tmp/tinyssh-release/usr/share/man/man1 (file does not exist)
make[1]: *** [Makefile:6: install] Error 111
make[1]: Leaving directory '/tmp/tinyssh-build'
make: *** [Makefile:30: build] Error 2
```

It looks like this is because the code creates manfiles directly in ${man}, but the install code expects to find it in man/man$n/. Digging in now to see if I can get a clean patch to fix it

--- a/make-install.sh
+++ b/make-install.sh
@@ -39,11 +39,12 @@ echo "=== `date` === finishing"
 confman="`head -1 conf-man`"
 mandir=`echo "$1/${confman}" | sed 's,//,/,g'`
 echo "=== `date` === installing man directory ${mandir}"
-mkdir -p "${bindir}" || exit 111
+mkdir -p "${mandir}" || exit 111
 ls "${man}" | sort |\
 while read x
 do
   n=`echo "${x}" | cut -d'.' -f2`
+  mkdir -p "${mandir}/man${n}" || exit 111
   "${installcmd}" "${man}/${x}" "${mandir}/man${n}" "${x}.tmp" "${x}" || exit 111
   echo "=== `date` ===   installing ${man}/${x} -> ${mandir}/man${n}/${x}"
 done || exit 111

